web/cobrands/bromley/layout.scss

```diff
@@ -1,135 +1,270 @@
     }
-    .sticky-sidebar {
-        left: 672px;
-        aside {
-            top:7em;
-        }
+    #site-logo {
+      width: 159px;
+      height: 114px;
     }
+  }
 }
-.ie6, .ie7 {
-    body.mappage .container {
-        width: 464px;
-        margin-left: 0;
-    }
+.ie6 body.frontpage #site-logo {
+  background:url(/cobrands/bromley/bromley-logo.png) 0 0 no-repeat;
 }
 
-#front-main {
-    @include border-radius(1em 1em 0 0);
-    margin: 2em 1em 0;
-    padding-top: 0;
+// Override the container width to match Bromley' site, which is wider
+.container {
+  width: 100%;
+  max-width: 1200px;
 }
 
-// Don't want gap that normal site has
-body.frontpage {
-    .table-cell {
-        .content {
-            margin-top: 0;
-        }
-    }
+// Override the FMS main menu to give it a background colour
+#main-menu {
+  background-color: $bromley_blue;
+}
+// Override the selected element to match. We need to do this separately
+// because of FMS's selectors being quite specific
+#main-nav ul#main-menu li span {
+  color: $nav_fg;
+  background-color: $nav_fg_hover;
+}
+
+// Override the "report a problem" button because it's not really a button
+#main-nav ul#main-menu li a.report-a-problem-btn {
+  color: $nav_fg;
+}
+
+// Fix bad margin on local alerts form
+body.alertindex form.full-width {
+  margin-left: 0;
 }
 
-// Don't want FixMyStreet logo on desktop, and we have a skip to content in
-// Bromley header
-#site-header {
-    display: none;
+// Override the little banner you get when you're logged in on the map page
+// because FMS makes it a different colour and moves it over to the left.
+body.mappage #user-meta p {
+  background-color: $bromley_blue;
+  border-bottom: 1px solid #444;
+  left: 2.65em;
 }
 
+// And also override the banner on other pages so that it lines up properly
 #user-meta {
-    display: none;
+  max-width: 1200px;
+}
+#user-meta p,
+body.frontpage #user-meta p {
+  top: 0;
+  right: auto;
+  background-color: $bromley_blue;
+  color: #fff;
+  box-shadow: none;
 }
 
-// Currently hiding, but do want some internal navigation somewhere at the end
-.nav-wrapper {
-    display: none;
+// Because we've moved the user-meta, we also have to move the #front-main div
+// down a bit so that it shows the #user-meta
+#front-main {
+  margin-top:3em
+}
+
+// The page h1 needs to be white on desktop, because it's in a blue box
+h1.main {
+  color: #fff;
 }
 
-// White background, so no shadow or margin needed.
-.content {
-    margin: 0;
-    @include box-shadow(none);
+// Because we've changed the page background, the footer looks a bit rubbish
+footer,
+body.twothirdswidthpage .container .content footer .tablewrapper,
+body.fullwidthpage .container .content footer .tablewrapper {
+  background-color: $bromley_blue;
 }
-.ie6, .ie7, .ie8  {
-    .content {
-        margin: 0;
-        border: none;
+
+// Make the powered by FMS logo work in Old IE. FMS fixes this in IE8, but
+// it's broken in IE6 and 7 too.
```

Comment:
Sure, I've moved it to sass/_layout.scss, I don't think there's any harm in doing that.